pytests/conftest.py

```diff
@@ -131,6 +133,13 @@ def check_valgrind(self):
             self.config['valgrind_disabled_reason'] = 'valgrind minimum version constraint not met'
             return
 
+        if 'valgrind_minimum_version_{}'.format(ARCH) in self.config:
+            minimum_version = self.version_str_to_int(self.config['valgrind_minimum_version_{}'.format(ARCH)])
```

Comment:
is there a indentation error here?

Reply:
No. Why? Outer if block (line 136) checks if `valgrind_minimum_version_ARCH` is set, otherwise we would get a val error in the next line.

Or are you confused because line 137 is too long? I should probably break it, like in 139/140.

Reply:
Yeah, my bad. Sorry. LGTM